src/firmware/main/app/world/wheel.h

```diff
@@ -1,18 +1,50 @@
 #pragma once
 
 /**
- * This struct represents a robot wheel, from the perspective of firmware
+ * This struct represents a robot wheel/motor, from the perspective of firmware
  */
 typedef struct Wheel Wheel_t;
 
+/**
+ * This struct holds wheel/motor constants
+ */
+typedef struct WheelConstants
+{
+    // The current per unit torque for the motor attached to this wheel [A/(n*m)]
```

Comment:
Uber Mega Nit:
Usually, newtons is capitalized (N)

Reply:
absolutely right, changed